x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts

```diff
@@ -44,7 +44,7 @@ async function getSpacesUsage(
   spacesAvailable: boolean
 ) {
   if (!spacesAvailable) {
-    return {} as UsageStats;
+    return null;
```

Comment:
**question** do you remember why this change was required? In my brief testing, the existing code doesn't appear to throw a TS or runtime error

Reply:
TypeScript complained that `available` and `enabled` props were rewritten here https://github.com/restrry/kibana/blob/c1d22c101cd300840cb8a03f4bb2b34d24c8698a/x-pack/plugins/spaces/server/usage_collection/spaces_usage_collector.ts#L152-L153
I checked the code and decided it's better to return `null` explicitly instead of manual casting since an empty object doesn't implement the `UsageStats` interface.

Reply:
Got it, thanks for explaining!